.github/workflows/pack-upload.yml

```diff
@@ -104,3 +104,26 @@ jobs:
           pwd
           yarn oclif upload tarballs
           ./scripts/upload/deb
+
+  pack-and-upload-windows:
+    runs-on: ubuntu-latest
```

Comment:
I would expect this to need to run on windows. Just a guess.

Reply:
Not necessarily. That is why we install NSIS below, I believe.